store/tikv/lock_resolver.go

```diff
@@ -243,7 +238,7 @@ func (lr *LockResolver) BatchResolveLocks(bo *Backoffer, locks []*Lock, loc Regi
 		metrics.LockResolverCountWithExpired.Inc()
 
 		// Use currentTS = math.MaxUint64 means rollback the txn, no matter the lock is expired or not!
-		status, err := lr.getTxnStatus(bo, l.TxnID, l.Primary, callerStartTS, math.MaxUint64, true, false, l)
+		status, err := lr.getTxnStatus(bo, l.TxnID, l.Primary, math.MaxUint64, math.MaxUint64, true, false, l)
```

Comment:
It seems a bit more reasonable to me to set 0 to `callerStartTS` when it is actually meaningless.

Reply:
We may add some new comments for the `getTxnStatus` as it's a bit complex to use now.